docs/shared-content/scripts/create-script-step-scripts.include.md

```diff
@@ -0,0 +1,158 @@
+```powershell
+
+# Define working variables
+$octopusURL = "https://youroctourl"
+$octopusAPIKey = "API-YOURAPIKEY"
+$header = @{ "X-Octopus-ApiKey" = $octopusAPIKey }
+$space = "Spaces-1"
+
+# Project details
+$projectName = "MyProject"
+
+try
+{
+    # Get project
+    $project = (Invoke-RestMethod -Method Get -Uri "$octopusURL/api/$space/projects/all" -Headers $header) | Where-Object {$_.Name -eq $projectName}
+
+    # Get deployment process
+    $deploymentProcess = (Invoke-RestMethod -Method Get -Uri "$octopusURL/api/$space/deploymentprocesses/$($project.DeploymentProcessId)" -Headers $header)
+
+    # Get current steps
+    $steps = $deploymentProcess.Steps
+
+    # Add new step to process
+    $steps += @{
+      Name = "Run a script"
+      Properties = @{
+        'Octopus.Action.TargetRoles' = "Test-Role"
+      }  
+      Condition = "Success"
+      StartTrigger = "StartAfterPrevious"
+      PackageRequirement = "LetOctopusDecide"
+      Actions = @(
+        @{
+            ActionType = "Octopus.Script"
+            Name = "Run a script"
+            Environments = @()
+            ExcludedEnvironments = @()
+            Channels = @()
+            TenantTags = @()
+            Properties = @{
+                'Octopus.Action.RunOnServer' = "false"
+                'Octopus.Action.EnabledFeatures' = ""
+                'Octopus.Action.Script.ScriptSource' = "Inline"
+                'Octopus.Action.Script.Syntax' = "PowerShell"
+                'Octopus.Action.Script.ScriptFilename' = $null
+                'Octopus.Action.Script.ScriptBody' = "Write-Host `"Hello world`""
```

Comment:
Should this example also have its content using `$ScriptBody` like the powershell client?